ecs-init/engine/engine.go

```diff
@@ -168,7 +170,14 @@ func (e *Engine) load(image io.ReadCloser, err error) error {
 // StartSupervised starts the ECS Agent and ensures it stays running, except for terminal errors (indicated by an agent exit code of 5)
 func (e *Engine) StartSupervised() error {
 	agentExitCode := -1
-	for agentExitCode != terminalSuccessAgentExitCode && agentExitCode != terminalFailureAgentExitCode {
+	const (
```

Comment:
please move constants to the top of the file

Reply:
I would prefer them here as there's no potential for them to be used anywhere else so no utility to the scope being package-wide

Reply:
:+1: to moving this to the outer scope, there's no need to _use_ them elsewhere in the package, but they also don't need to be in this scope (and it'll compile to roughly the same code). Please add these to the top with some comments for tuning later.

Reply:
We've typically put all constants at the top as a style choice. It runs a bit against Effective Go's style recommendations, but its more or less what we've settled on.

FWIW, we also break their guidelines with variable names, where we've always prefered full words.

Reply:
`minTime` would need to become something like `minServiceStartRetryBackoffTime` to describe what a package-level constant is doing....is this preferable?

Reply:
I've gone ahead and made them package-level constants